recipes/soci/all/conanfile.py

```diff
@@ -0,0 +1,154 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class SociConan(ConanFile):
+    name = "soci"
+    homepage = "https://github.com/SOCI/soci"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "The C++ Database Access Library "
+    topics = ("C++", "database-library", "oracle", "postgresql", "mysql", "odbc", "db2", "firebird", "sqlite3", "boost" )
+    license = "Boost"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    options = {
+        "fPIC":       [True, False],
+        "static":     [True, False],
```

Comment:
```suggestion
```

Shared is the official option in Conan: https://docs.conan.io/en/latest/reference/conanfile/attributes.html#options

Don't mix both on same package. A package should provide OR static OR shared libraries only.